packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -101,16 +101,18 @@ export function Modal({
     }
     setView(view);
   };
+
   const modalOverlayClassName = classNames(
     'overlay fixed inset-0 z-modal flex flex-col items-center bg-overlay-quaternary-onion',
     modalKindAndSizeToOverlayClassName[kind]?.[size],
     modalKindToOverlayClassName[kind],
     overlayClassName,
   );
   const modalClassName = classNames(
-    'modal relative flex max-w-full flex-col items-center rounded-16 border border-theme-divider-secondary bg-accent-pepper-subtlest antialiased shadow-2 focus:outline-none',
+    'modal relative flex max-w-full flex-col items-center border border-theme-divider-secondary bg-accent-pepper-subtlest antialiased shadow-2 focus:outline-none',
+    'h-full tablet:h-auto tablet:rounded-16',
     modalKindToClassName[kind],
-    modalSizeToClassName[size],
+    isMobile ? 'w-full' : modalSizeToClassName[size],
```

Comment:
We could do this with css media queries, no?

Reply:
in this case, I'm echoing @kopancek on using media queries

Reply:
Right. Yeah, we can convert this.